package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "aws-connect-sso-web-app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint",
+    "format": "prettier --check --ignore-path .gitignore .",
+    "format:fix": "prettier --write --ignore-path .gitignore .",
+    "test:e2e": "cypress open --e2e --browser=chrome",
+    "test:e2e:headless": "cypress run --e2e --browser=chrome"
+  },
+  "dependencies": {
+    "jsonwebtoken": "9.0.2",
+    "next": "13.5.4",
+    "node-fetch": "2.7.0",
```

Comment:
What version of node is this on? Don't the later versions have fetch natively implemented?

Reply:
I was using 18 in which built-in `fetch` is still experimental, so I didn't want to commit to it yet.

Reply:
Welp... in the newest frontend repo, I should probably upgrade to Node 20 anyway!
It is now upgraded and `node-fetch` has been removed, thanks for the suggestion!